Yosemite/YosemiteTests/Stores/ShippingLabelStoreTests.swift

```diff
@@ -823,8 +823,8 @@ final class ShippingLabelStoreTests: XCTestCase {
         // Given
         let mockAddress = ShippingLabelAddress.fake()
         let mockPackages = [ShippingLabelPackagePurchase.fake()]
-        let expectedLabel = ShippingLabel.fake().copy(shippingLabelID: 13579, status: .purchaseInProgress)
-        let labelStatusResponse = ShippingLabelStatusPollingResponse.purchased(expectedLabel)
+        let inProgressLabel = ShippingLabel.fake().copy(shippingLabelID: 13579, status: .purchaseInProgress)
+        let labelStatusResponse = ShippingLabelStatusPollingResponse.purchased(inProgressLabel)
```

Comment:
The rename was a nitpick, really.

To me "expected" is the name to use for a value to run expectation/assertions against, e.g.

```swift
let expected = 4
XCTAssertEqual(mySumImplementation(3, 1), expected)
```

In this usage, the `ShippingLabel` value is not expected but part of the test step (also called "given", or, my preferred, arrange stage)